### PR TITLE
Add qa question about a11y tools

### PR DESCRIPTION
**Pull request by YuriDevAT**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by jdwilkin4** [comment]
If this category is approved, then this question will need to be moved to the accessibility quiz not the QA one.

**Comment by YuriDevAT** [comment]
Yep, got it 👍🏽  Thanks!

File: src/data/quality-assurance-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,14 @@
 const qualityAssuranceQuiz = [
+  {
+    Question: "What are web accessibility evaluation tools?",
+    Answer: "Software programs or online services that help determine if web content meets accessibility standards.",
+    Distractor1: "Tools that help determine if unit tests are well written.",
+    Distractor2: "Software programs that help determine if a website is secure.",
+    Distractor3: "Online services that help determine if customer satisfaction is high.",
+    Explanation:
+      "Web accessibility evaluation tools are software programs or online services that help determine if web content meets accessibility standards.",
```
Comment (jdwilkin4) [inline src/data/quality-assurance-quiz.ts:9]:
```suggestion
      "Web accessibility evaluation tools are programs used to ensure that software adheres to accessibility guidelines as laid out by the W3C.",
```

File: src/data/quality-assurance-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+    Link: "https://www.w3.org/WAI/test-evaluate/tools/",
+  },
```
Comment (jdwilkin4) [inline src/data/quality-assurance-quiz.ts:11]: This question looks good to me.
Quincy approved the new accessibility category.
I just pushed up a change to add that new category.

Please move this question over to the accessibility quiz. 👍

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by jdwilkin4** [comment]
I am closing this PR because it looks like you opened up a new one

**Comment by YuriDevAT** [comment]
Thanks @jdwilkin4.

**Pull request closed without merging**